config.map

```diff
@@ -3,7 +3,7 @@ SCRAM_ARCH=slc7_amd64_gcc12;PKGTOOLS_TAG=V00-34-XX;CMSDIST_TAG=IB/CMSSW_14_0_X/m
 SCRAM_ARCH=el8_amd64_gcc12;PKGTOOLS_TAG=V00-34-XX;CMSDIST_TAG=IB/CMSSW_14_0_X/master;RELEASE_BRANCH=master;RELEASE_QUEUE=CMSSW_14_0_DBG_X;DOCKER_IMG=cmssw/el8;BUILD_HOUR=23;BUILD_DAY=4;
 SCRAM_ARCH=el8_amd64_gcc12;PKGTOOLS_TAG=V00-34-XX;CMSDIST_TAG=IB/CMSSW_14_0_X/master;RELEASE_BRANCH=master;RELEASE_QUEUE=CMSSW_14_0_NONLTO_X;DOCKER_IMG=cmssw/el8;PR_TESTS=1;BUILD_OPTS=no-lto;
 SCRAM_ARCH=el8_amd64_gcc12;PKGTOOLS_TAG=V00-34-XX;CMSDIST_TAG=IB/CMSSW_14_0_X/cudart;RELEASE_BRANCH=master;RELEASE_QUEUE=CMSSW_14_0_CUDART_X;DOCKER_IMG=cmssw/el8;BUILD_HOUR=0,23;BUILD_DAY=0,3;
-SCRAM_ARCH=el8_amd64_gcc12;PKGTOOLS_TAG=V00-34-XX;CMSDIST_TAG=IB/CMSSW_14_0_X/master;RELEASE_BRANCH=master;RELEASE_QUEUE=CMSSW_14_0_SKYLAKEAVX512_X;DOCKER_IMG=cmssw/el8;BUILD_HOUR=23;BUILD_DAY=0,4;
+SCRAM_ARCH=el8_amd64_gcc12;PKGTOOLS_TAG=V00-34-XX;CMSDIST_TAG=IB/CMSSW_14_0_X/master;RELEASE_BRANCH=master;RELEASE_QUEUE=CMSSW_14_0_SKYLAKEAVX512_X;DOCKER_IMG=cmssw/el8;BUILD_HOUR=23;BUILD_DAY=0,4;ADDITIONAL_TESTS=igprof-pp,profiling
```

Comment:
@gartung , please add the `;` at the end

Reply:
Done